### DISCUSSION TITLE
Make it possible to leave "deferred comments" which are not attached to a particular line of code

### DISCUSSION THREAD

**Discussion by nikitabobko** [community/community#7359] [Pull Requests]
Definition (I don't know how it's called and failed to find it in GitHub documentation so I will define it myself): "deferred comment" is a Pull Request review comment which is only published once you hit "Submit review" button. Multiple "deferred comments" are published altogether at once.

It would be cool to be able to leave "deferred comments" on the "Conversation" Pull Requests tab.

Right now, it's possible to leave "deferred comments" only if it's attached to some line of code but there are cases when I want to comment on the whole PR in general

**Comment by tonglil** [comment]
I'm curious if you can add your "deferred comments" into the review comment box. AFAIK what you type there is persisted until you hit "submit"?

Related but separate, I want the "main review comment" and other "non-review, non-file based comments" to be reply-able (create a thread (see https://github.com/orgs/community/discussions/11674).
